src/collections/_documentation/development/sdk-dev/unified-api.md

```diff
@@ -188,7 +206,8 @@ Why not just have a `get_current_scope()` function instead of this indirection?
 
 - `scope.add_error_processor(processor)` (optional): Registers an error processor function.  It takes an event and exception object and returns a new event or `None` to drop it.  This can be used to extract additional information out of an exception object that the SDK cannot extract itself.
 
-- `scope.clear()`: Resets a scope to default values (prevents inheriting).  This never clears registered event processors.
+- `scope.clear()`: Resets a scope to default values however keeping all 
```

Comment:
This is confusing, perhaps: "Resets a scope to default values while keeping registered event processors, which does not affect child or parent scopes."